frontend/javascripts/oxalis/view/components/border_toggle_button.js

```diff
@@ -23,7 +30,7 @@ function BorderToggleButton(props: Props) {
     inFooter === true ? "footer-button" : "flexlayout__tab_toolbar_button"
   }`;
   const imageSrc = `/assets/images/icon-sidebar-${iconKind}-${side}-${
```

Comment:
@philippotto If possible, i would try to avoid the JS code knowing about the selected theme, because it will not update reactively when the theme is changed. In this case, the image could be implemented as a background image.

Reply:
ah, good idea with the background image :+1: `--is-dark-mode` felt clever, but using a pure bg image is obviously better. i will change it now